agent/tool-scripts/pprof

```diff
@@ -146,7 +160,12 @@ ose_pprof() {
 
 collect_data() {
     while true; do
-            $tool_bin tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_.txt http://localhost:6060/debug/pprof/profile
+        if [ $(systemctl list-unit-files | grep atomic-openshift-master.service | awk {'print $2'}) == "masked" ]; then
+            $tool_bin tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_controller.txt http://localhost:6060/debug/pprof/profile &
```

Comment:
Have you tested this?  I am not sure I see how this works.  The start-tools will create a sub-shell and then immediately exit.  The when a stop-tools comes along, how will it find the sub-processes that were created?  Instead, I think we have to wait for a these processes that we create, and have the script detect when it is "killed" and in turn kill the subprocesses created here.